build/travis/byte_mag_latest_before_script.sh

```diff
@@ -0,0 +1,44 @@
+# Installation for latest magento mirror
```

Comment:
no it does not, it's basically the same setup but instead of using magerun to download it, it downloads byte's latest tar file, extracts it and tests against it. It could be extended / merged with the current when but then you have to catch the environment variable and if it's the byte mirror, do a slightly different thing.